cmd/convert.go

```diff
@@ -46,6 +46,7 @@ var convertCmd = &cobra.Command{
 			ToStdout:               ConvertStdout,
 			CreateChart:            ConvertChart,
 			GenerateYaml:           ConvertYaml,
+			GenerateJson:           ConvertJson,
```

Comment:
@surajssd feels cleaner to have both, right? json + yaml generation (by default, you can set the value of GenerateYaml to `true` in cobra)